posts/2020-12-31-Rust-1.49.0.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: post
+title: "Announcing Rust 1.49.0"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.49.0. Rust is a
+programming language that is empowering everyone to build reliable and
+efficient software.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.49.0 is as easy as:
+
+```console
+rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.49.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1490-2020-12-31
+
+## What's in 1.49.0 stable
+
+For this release, we have some new targets and an improvement to the test
+framework. See the [detailed release notes][notes] to learn about other
+changes not covered by this post.
+
+### 64-bit ARM Linux reaches Tier 1
+
+The Rust compiler supports [a wide variety of targets][platform-support], but
+the Rust Team can't provide the same level of support for all of them. To
+clearly mark how supported each target is we created a tiering system:
+
+* Tier 3 targets are technically supported by the compiler, but we don't check
+  whether their code build or passes the tests, and we don't provide any
+  prebuilt binaries as part of our releases.
+* Tier 2 targets are guaranteed to build and we provide prebuilt binaries, but
+  we don't execute the test suite on those platforms: the produced binaries
+  might not work or might have bugs.
+* Tier 1 targets provide the highest support guarantee, and we run the full
+  suite on those platforms for every change merged in the compiler. Prebuilt
+  binaries are also available.
```

Comment:
This is already in the platform-support page, right? Does it make sense to repeat it in the blog? I think the following paragraph ('highest guarantees') is good enough to understand why this is a big deal.

Reply:
I'd prefer to keep it to provide more context for people who are not already familiar with our policy and who are going to miss the link.